API.yaml

```diff
@@ -1430,6 +1510,50 @@ components:
           description: The bookmark's unique ID, generated by the server.
           type: integer
           format: int128
+    Annotation:
```

Comment:
Annotation need an entryId/seriesId and a y range in case of XY. Is it possible to generalize this concept?